services/media/workflows/package.json

```diff
@@ -27,7 +27,7 @@
   ],
   "dependencies": {
     "@apollo/client": "^3.7.9",
-    "@axinom/mosaic-ui": "0.33.2",
+    "@axinom/mosaic-ui": "0.34.0-rc.33",
```

Comment:
We shouldn't use RC versions

Reply:
Changes to package version update removed during rebase.